Include/object.h

```diff
@@ -142,7 +147,7 @@ struct _object {
 #endif
 
     PyTypeObject *ob_type;
-};
+} Py_ALIGNED(1 << _PyObject_ALIGNMENT_SHIFT);
```

Comment:
We should not be adding alignment attributes to `Include/object.h`. Limit them to `PyGC_Head` where they are needed: `PyGC_Head` has stricter alignment requirements than it's natural alignment on m68k.